packages/components-native/src/Banner/types.ts

```diff
@@ -29,22 +29,29 @@ export interface BannerProps {
   readonly action?: ActionProps;
 
   /**
-   *  Optional custom content that can be displayed below the text and details
+   * Custom content to be displayed in the banner
    */
-  readonly children?: ReactNode;
+  readonly children?: React.ReactElement | string;
 
   /**
-   * Text to display below the 'text' prop. Will display a single message or bullet points of messages if multiple are provided
+   * **Deprecated**: Use `children` with a `<TextList level="textSupporting" />` instead
+   * @deprecated
    */
   readonly details?: string[];
 
   /**
-   * The primary text to display in the banner
+   * **Deprecated**: Use `children` instead
```

Comment:
```suggestion
   * **Deprecated**: Use `children`with a `<Text level="textSupporting" />` or a `string` instead
```
Minor suggestion. May need to do locally in case of a lint error

Reply:
Nice catch - done here https://github.com/GetJobber/atlantis/pull/1473/commits/db39a634c619ef8d3a54e54b3982597febf41543